owner-onboarding-server/src/main.rs

```diff
@@ -423,32 +432,48 @@ async fn main() -> Result<()> {
                 .or(handler_to2_prove_device)
                 .or(handler_to2_device_service_info_ready)
                 .or(handler_to2_device_service_info)
-                .or(handler_to2_done),
+                .or(handler_to2_done)
+                .or(ov_filter(user_data.clone())),
         )
         .recover(fdo_http_wrapper::server::handle_rejection)
         .with(warp::log("owner-onboarding-service"));
 
-    log::info!("Listening on {}", bind_addr);
-    let server = warp::serve(routes);
-
-    let maintenance_runner =
-        tokio::spawn(async move { perform_maintenance(user_data.clone()).await });
-
-    let server = server
-        .bind_with_graceful_shutdown(bind_addr, async {
-            signal(SignalKind::terminate()).unwrap().recv().await;
-            log::info!("Terminating");
-        })
-        .1;
-    let server = tokio::spawn(server);
-
-    tokio::select!(
-    _ = server => {
-        log::info!("Server terminated");
-    },
-    _ = maintenance_runner => {
-        log::info!("Maintenance runner terminated");
```

Comment:
This handles the core functionality of the server according to the FDO standard, we cannot delete it.